### DISCUSSION TITLE
vscode plugin

### DISCUSSION THREAD

**Discussion by mturilin** [ngs-lang/ngs#648] [Ideas]
I would love to have a plugin to edit NGS in vscode with syntax highlighting

**Comment by ilyash-b** [comment]
We had some alpha version of this somewhere. @organom might know where it is

**Comment by organom** [comment]
@ilyash-b under https://github.com/ngs-lang/ngs-syntax-vscode

**Reply by ilyash-b** [reply]
@mturilin , wanna try what @organom said?
